test/Test/Smoke.hs

```diff
@@ -1,21 +1,18 @@
-{- SPDX-FileCopyrightText: 2019 Bitcoin Suisse
+{- SPDX-FileCopyrightText: 2020 Bitcoin Suisse
```

Comment:
Boy, this PR took so long the copyright year needs to be changed twice :sweat_smile: 

Jokes aside this repo has a lot (mostly I think) 2019 there, and I suppose others are in a similar condition :thinking: 
Looks like new issues are coming.